src-data/experiment/hist-nofire.json

```
@@ -0,0 +1,38 @@
+{
+    "id": "hist-nofire",
+    "type": [
+        "wcrp:experiment",
+        "cmip7"
+    ],
+    "activity": [
+        "FireMIP / ID31"
+    ],
+    "description": "This experiment is similar to the esm-hist experiment in DECK, but it features no fire globally throughout the historical period and is initialized from a no-fire spin-up.",
+    "label": "hist-noFire",
+    "long-label": "no fires occur globally",
+    "minimum-number-of-years": "172",
+    "model-realms": [
+        {
+            "id": "aogcm",
+            "is-required": true
+        },
+        {
+            "id": "bgc",
+            "is-required": true
+        },
+        {
+            "id": "aer",
+            "is-required": true
+        },
+        {
+            "id": "chem",
+            "is-required": false
+        }
+    ],
+    "parent-experiment": [
+        "esm-hist"
```

The following has been agreed on #10

```suggestion
        "esm-piControl"
```